local-cluster/tests/local_cluster.rs

```diff
@@ -386,7 +386,7 @@ fn test_restart_node() {
     );
     cluster.exit_restart_node(&nodes[0], validator_config, SocketAddrSpace::Unspecified);
     cluster_tests::sleep_n_epochs(
-        0.5,
```

Comment:
why does this need to increase?

Reply:
this test is bit flaky, will have to visit the test separately.